build-utils/integration-docs-fetch-plugin.ts

```diff
@@ -63,8 +68,32 @@ class IntegrationDocsFetchPlugin {
     }
   }
 
-  apply(compiler) {
-    compiler.hooks.beforeRun.tapAsync('IntegrationDocsFetchPlugin', fetch.bind(this));
+  fetch: Parameters<webpack.Compiler['hooks']['beforeRun']['tapAsync']>[1] = (
```

Comment:
Made this an actual method of the plugin instead of doing this wierd `.call` stuff